### PR TITLE
Very basic browse

### PR DESCRIPTION
**Pull request by rjaquino**
Shows tag results, and recent results if no tags selected

### CODE REVIEW

File: r/model/upload.py [hunk 1]
```diff
@@ -61,12 +61,26 @@ def get_by_slug(slug: str):
...
 def uploads_for_feed(user_id):
     uploads = r.model.upload.Upload.get_where({'user_id': user_id}, associations=['files', 'files.comments', 'tag_uploads.tag', 'user'], order='creation_time DESC')
+    return _prepare_upload_stories(uploads)
+
+def uploads_for_browse(tag_ids):
+    # if none provided, fetch recents
+    if len(tag_ids) == 0:
+        uploads = r.model.upload.Upload.get_where(associations=['files', 'files.comments', 'tag_uploads.tag', 'user'], order='creation_time DESC', limit=20)
+        return _prepare_upload_stories(uploads)
+
```
Comment (tmacwill) [inline r/model/upload.py]: extra newline?

**Comment by tmacwill** [comment]
can't merge :\

**Comment by tmacwill** [comment]
ping

**Comment by rjaquino** [comment]
Will fix when you're downloading mewtwo

On Wed, Apr 15, 2015 at 12:12 PM, Tommy MacWilliam <notifications@github.com
> wrote:

> ping
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tmacwill/review/pull/3#issuecomment-93536024>.
>



-- 
- R.J.

**Pull request closed without merging**